Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -340,32 +334,42 @@ extension ActivityStreamPanel {
     }
 
     @objc private func longPress(longPressGestureRecognizer: UILongPressGestureRecognizer) {
-        if longPressGestureRecognizer.state == UIGestureRecognizerState.Began {
-            let touchPoint = longPressGestureRecognizer.locationInView(self.view)
-            if let indexPath = tableView.indexPathForRowAtPoint(touchPoint) {
-                if Section(indexPath.section) == .Highlights {
-                    presentContextMenu(history[indexPath.row], atIndexPath: indexPath)
-                } else {
-                    let topSiteCell = self.tableView.cellForRowAtIndexPath(indexPath) as! ASHorizontalScrollCell
-                    let touchPointWithinTopSiteCell = longPressGestureRecognizer.locationInView(topSiteCell.collectionView)
-                    if let indexPath = topSiteCell.collectionView.indexPathForItemAtPoint(touchPointWithinTopSiteCell) {
-                        presentContextMenu(topSites[indexPath.row], ofCollectionView: topSiteCell.collectionView, atIndexPath: indexPath)
-                    }
-                }
+        guard longPressGestureRecognizer.state == UIGestureRecognizerState.Began else { return }
```

Comment:
@farhanpatel , are you referring to [this rule](https://github.com/raywenderlich/swift-style-guide#spacing)?